datascience-notebook/onbuild/Dockerfile

```diff
@@ -0,0 +1,8 @@
+from jupyter/datascience-notebook
+
+
+ONBUILD COPY requirements.txt /usr/src/app/
+
+ONBUILD RUN pip install --no-cache-dir -r requirements.txt
+
+ONBUILD COPY . /home/jovyan/work
```

Comment:
I wonder if it would be better to do this in place of the previous copy. That way `requirements.txt` can just be used.